resources/monitoring/templates/prometheus/kyma-rules/prometheus.rules.yaml

```diff
@@ -1,19 +0,0 @@
-apiVersion: monitoring.coreos.com/v1
-kind: PrometheusRule
-metadata:
-  name: {{ printf "%s-%s" (include "kube-prometheus-stack.fullname" .) "kyma-prometheus.rules" | trunc 63 | trimSuffix "-" }}
-  labels:
-    app: {{ template "kube-prometheus-stack.name" . }}
-{{ include "kube-prometheus-stack.labels" . | indent 4 }}
-spec:
-  groups:
-  - name: prometheus.rules
-    rules:
-    - alert: ScrapeLimitForTargetExceeded
```

Comment:
We will need that rule to inform the user about reaching the limit

Reply:
Indeed, thanks I'll add it back.